src/tune-hyperparameters/src/main/scala/Hyperparams.scala

```diff
@@ -0,0 +1,88 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.ml.spark
+
+import org.apache.spark.ml.classification._
+import org.apache.spark.ml.param.Param
+
+/**
+  * Provides good default hyperparameter ranges and values for sweeping.
+  * Publicly visible to users so they can easily select the parameters for sweeping.
+  */
+object Hyperparams {
```

Comment:
Is it possible to use the tuner on other models than these?

Reply:
Yes!  These are just defaults that the user can choose conveniently.  The user can build any hyperparams they wish.

Reply:
Okay can we rename this object to something like DefaultHyperparameters then?

Reply:
done!